ffi/derive/tests/ui_fail/derive_skip_struct.rs

```diff
@@ -1,12 +1,14 @@
 use core::mem::MaybeUninit;
 
 use getset::{MutGetters, Setters};
-use iroha_ffi::{ffi_export, IntoFfi, TryFromFfi, TryFromReprC};
+use iroha_ffi::{ffi_export, IntoFfi, TryFromReprC};
 
-#[derive(Clone, Setters, MutGetters, IntoFfi, TryFromFfi)]
-#[ffi_export]
+/// FfiStruct
+#[derive(Clone, Setters, MutGetters, IntoFfi, TryFromReprC)]
 #[getset(skip)]
```

Comment:
What does `#[getset(skip)]` mean for a struct and why is this bad? Quickly looking at [getset doc](https://docs.rs/getset/latest/getset/index.html) I can't find this info

Reply:
it's not bad. You can use `getset(skip)`. This test confirms that if you skipped the derivation of getter for a field the corresponding extern function wasn't generated for it